code/modules/antagonists/revolution/enemy_of_the_state.dm

```diff
@@ -0,0 +1,68 @@
+
+/**
+ * When the station wins, any remaining living headrevs become Enemies of the State, a small solo antagonist.
+ * They either have the choice to fuck off and do their own thing, or try and regain their honor with a hijack.
+ */
+/datum/antagonist/enemy_of_the_state
+	name = "Enemy of the State"
+	show_in_antagpanel = FALSE
+	show_name_in_check_antagonists = TRUE
+	hijack_speed = 2 //not like they have much to do
+
+/datum/antagonist/enemy_of_the_state/proc/forge_objectives()
+
+	var/datum/objective/exile/exile_choice = new
+	exile_choice.owner = owner
+	exile_choice.objective_name = "Choice"
+	objectives += exile_choice
+
+	var/datum/objective/hijack/hijack_choice = new
+	hijack_choice.owner = owner
+	hijack_choice.objective_name = "Choice"
+	objectives += hijack_choice
+
+/datum/antagonist/enemy_of_the_state/on_gain()
+	//needs to sleep for a tick so the old antag datum can begone before it announces objectives.
+	sleep(1)
```

Comment:
:-1: Don't do this in on_gain, randomly yielding code is terrible. Just do it wherever the actual old antag datum is removed.